src/lib.rs

```diff
@@ -1,5 +1,6 @@
 pub mod common;
 pub mod decimal;
 pub mod error;
+pub mod ops_traits;
```

Comment:
Nit:
```suggestion
#[cfg(feature = "ops-traits")]
pub mod ops_traits;
```
Then you can remove all other `cfg` in `ops_traits.rs` since the whole file will be disabled with just this one.